### PR TITLE
Adds celery beat to LDAP and plugin.

### PR DESCRIPTION
**Pull request by jvanderaa**
- Adds Celery Beat for LDAP and Plugin examples.

### CODE REVIEW

File: docker-compose.ldap.yml [hunk 1]
```diff
@@ -65,6 +65,24 @@ services:
       - "./local.env"
     tty: true
   # ---------------------------------
+  # Celery beat must only be used in > 1.2.0 installs. If setting to a previous install, remove this worker
+  # ----------------------------------
+  celery_beat:
+    image: "companyname/nautobot-ldap"
+    entrypoint: "sh -c 'nautobot-server celery beat -l $$NAUTOBOT_LOG_LEVEL'"
+    healthcheck:
```
Comment (nniehoff) [inline docker-compose.ldap.yml:73]: there is no health check for beat at this time

**Review by nniehoff — CHANGES_REQUESTED** [review]

File: docker-compose.ldap.yml (hunk 1 above)
Comment (jifox) [reply docker-compose.ldap.yml:73]: Maybe:  `restart: "unless-stopped"`   will be good too?

**Review by jdrew82 — APPROVED** [review]
LGTM! I believe I have this covered in #68 too.

**Comment by dylanbob** [comment]
Hi. This was approved 3 months ago, could this please be merged so as to clarify which elements should be included in the docker compose files, thus solving issue [#46](https://github.com/nautobot/nautobot-docker-compose/issues/46)? @jdrew82

**Comment by jdrew82** [comment]
I believe this can be closed as Beats is now part of the LDAP and other environments in the project.

**Comment by dylanbob** [comment]
Thanks for the (very) quick merge, I see many things changed

**Comment by jdrew82** [comment]
> Thanks for the (very) quick merge, I see many things changed

Yes, there was a bit of rearchitecting of the project to utilize Poetry and invoke to hopefully simplify usage of the project. Please let me know if there's anything that confusing or wrong in the documentation and we can get that corrected.

**Pull request closed without merging**